ui/app/send-v2.js

```diff
@@ -552,6 +553,47 @@ SendTransactionScreen.prototype.addToAddressBookIfNew = function (newAddress) {
   }
 }
 
+SendTransactionScreen.prototype.getEditedTx = function () {
+  const {
+    from: {address: from},
+    to,
+    amount,
+    gasLimit: gas,
+    gasPrice,
+    selectedToken,
+    editingTransactionId,
+    unapprovedTxs,
+  } = this.props
+
+  const editingTx = unapprovedTxs[editingTransactionId]
+
+  editingTx.txParams = {
+    from: ethUtil.addHexPrefix(from),
+    gas: ethUtil.addHexPrefix(gas),
+    gasPrice: ethUtil.addHexPrefix(gasPrice),
+  }
+
+  if (selectedToken) {
+    const data = '0xa9059cbb' + Array.prototype.map.call(
```

Comment:
what's `'0xa9059cbb'`?

Reply:
It is a signature associated with the ERC20 token transfer function. It gets added to the data property after calling token.transfer(). I have replaced it with a constant in ed39f5a8